tests/ui/consts/const-unstable-intrinsic.stderr

```diff
@@ -25,6 +25,10 @@ LL |         unstable_intrinsic::size_of_val(&x);
    |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    |
    = help: add `#![feature(unstable)]` to the crate attributes to enable
+help: add `#![feature(unstable)]` to the crate attributes to enable
```

Comment:
The duplicate message here does not seem nice?

Reply:
Fixed. There was a stray `#[help]` in the error type definition.